### DISCUSSION TITLE
kbd shortcuts

### DISCUSSION THREAD

**Discussion by giquo** [Taiko2k/Tauon#901] [Q&A] [answered]
Hi, put this on a discussion rather than a issue bcus I might be doing something wrong.

I LOVE the radio random feature, and I would like to use it more and quickly more often
I see that the shortcut is `"/"` buuuuuuuuut I haven't managed to used it ever :(

I got an ISO Keychron K8 (TKL kbd) and the `"/" `is accesible with `shift + 7`, when I press that combi, nothing happends :C

uuuuumm..... what options do we have? is the shortcuts restringed to ANSI keyboards? or 100% keyboards?
ty, and again, ty for your great work with this player, I love it very much

**Comment by giquo** [comment]
Just thought of this solutin after posting it (if is not implemented)
Like... what I would do would be something like:
1. is there a way to get which language keyboard is the user using?

So with that, auto-determine a set of keyboard shortcuts depending of that

**Comment by Taiko2k** [comment, marked as answer]
Not sure why it might not be working, isn't "/" usually the same as "?"?

In any case, Tauon actually has a way to re-assign keyboard shortcuts. Go to MENU > Settings > Open data folder. In that folder is a file named `input.txt`, edit that file, see the entry `random-track-start`. Save then restart Tauon.

**Reply by giquo** [reply]
https://preview.redd.it/7cfvisietpe21.png?width=830&format=png&auto=webp&s=30f759a7857e55400cfa400e4ba6d5dd3f4f3601
This is my kbd layout, is latin america spanish.

But I can carry with that answer, seems great to me.
Also... is that specified in the documentation? so I can completely feel embarrassed for not correct reading XD

**Reply by giquo** [reply]
well, after trying it with the layout I show in the picture
put `Shift 7` and didn't worked
ended up putting `{` for `random-track-start` and worked, I'll carry with that
thanks ^^ 

PD: the hint in the playback menu still shows `/` but eh... tsk tsk, is not big deal

**Comment by Solarunit** [comment]
>This is my kbd layout, is latin america spanish.

But that's not a TKL keyboard btw :)
